src/hotspot/share/cds/filemap.hpp

```diff
@@ -208,10 +208,11 @@ class FileMapHeader: private CDSFileMapHeaderBase {
   // The following fields are all sanity checks for whether this archive
   // will function correctly with this JVM and the bootclasspath it's
   // invoked with.
+public:
```

Comment:
Again this looks like it should be private with either a public accessor or else a friend declaration for the client code.

Reply:
same reason as above.